packages/default/scss/floating-label/_layout.scss

```diff
@@ -75,6 +75,7 @@
                 > .k-label {
                     left: auto;
                     right: $floating-label-offset-x;
+                    pointer-events: none;
```

Comment:
This is redundant, as the label will receive `pointer-events: none` from the `.k-state-empty > .k-label` selector on lines 40-45. Thus, this line can be removed.